aclk/aclk.c

```diff
@@ -1204,3 +1214,38 @@ char *ng_aclk_state_json(void)
     json_object_put(msg);
     return str;
 }
+
+void add_aclk_host_labels(void) {
+    DICTIONARY *labels = localhost->host_labels;
+
+#ifdef ENABLE_ACLK
+    rrdlabels_add(labels, "_aclk_ng_available", "true", RRDLABEL_SRC_AUTO|RRDLABEL_SRC_ACLK);
```

Comment:
Is there a special motive to have this call for `rrdlabels_add` here instead next group of `ifdef ENABLE_ACLK`?

Reply:
Considering that both calls will change the third argument only, I suggest to use the `ifdef` to define only it.

Reply:
thanks I changed it. I only moved this function from file to file without checking the code.